client/signup/steps/domains/index.jsx

```diff
@@ -676,7 +690,15 @@ export class RenderDomainsStep extends Component {
 					{ domainCart }
 				)
 			);
-			this.props.goToNextStep();
+
+			const productToRemove = this.props.cart.products.find(
```

Comment:
Since we only execute this code on multi-domains context, it seems safe. Otherwise, I'd put that `removeProductFromCart` in a conditional statement for the multi-domains feature.

Reply:
If we're not able to enable this feature for all flows are we better off moving this into its own step and including that step only in the flows that want multi domain selection?